Fixes #2501: Replace CNTT with Anuket (#2522)
cntt-n/CNTT: CONTRIBUTING.md

<a name="permissions"></a>
### Permission levels

> There are different permissions (levels) available for contributing into Anuket CNTT:

Reviewer: ```suggestion
There are different permissions (levels) available for contributing to Anuket:
```

<a name="labels"></a>
### Labels

> Every Issue and PR should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct Anuket CNTT processes.

Reviewer: ```suggestion
Every Issue and Patch Request (PR) should be tagged with one or more of the following labels to indicate their status and for the automation pipelines to apply the correct Anuket processes.
```

<!--

> * [Contribution Guidelines](https://github.com/cntt-n/Anuket CNTT/wiki/Contribution-Guidelines)

Reviewer: You can't do this -- there is no such link

> * [Approval Process](https://github.com/cntt-n/Anuket CNTT/wiki/Approval-Process) 

Reviewer: You can't do this -- there is no such link
